FRCaptchaEnterprise/FRCaptchaEnterprise/Sources/CaptchaConstant.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  CBConstants.swift
+//  FRCaptchaEnterprise
+//
+//  Copyright (c) 2024 ForgeRock. All rights reserved.
+//
+//  This software may be modified and distributed under the terms
+//  of the MIT license. See the LICENSE file for details.
+//
+
+import Foundation
+import FRAuth
+
+//  MARK: - CaptchaEnterpriseCallback
+
+public class CaptchaConstant {
+    static let type: String = "type"
```

Comment:
Maybe add here the ` "com.google.recaptcha"` and "INVALID_CAPTCHA_CLIENT" to use in the 

 let error = RecaptchaError(domain: "com.google.recaptcha",
                                          code: .errorCodeInternalError,
                                          message: "INVALID_CAPTCHA_CLIENT")
        self.setClientError("\(error.errorCode)\(error.description)")